.github/workflows/run-test-harness.yml

```diff
@@ -14,5 +14,3 @@ jobs:
         with:
           sdks-to-test: php
           sdk-github-sha: ${{github.event.pull_request.head.sha}}
-          github-token: ${{ secrets.TEST_HARNESS_GH_SECRET }}
```

Comment:
why is this removed?

Reply:
not used anymore - we have it using the built in auth.